src/drivers/tap_esc/tap_esc.cpp

```diff
@@ -247,6 +247,8 @@ TAP_ESC::init()
 	ConfigInfoBasicRequest   &config = packet.d.reqConfigInfoBasic;
 	memset(&config, 0, sizeof(ConfigInfoBasicRequest));
 	config.maxChannelInUse = _channels_count;
+	/* Enable closed-loop control */
+	config.controlMode = 1;
```

Comment:
> I thought that was a there until you had a chance to figure it out. No?

Final hw came out. I still get nothing useful back.  Could be something in the ESC's firmware... I know for example that the FW simply ignores commands to the LEDs.